analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -527,4 +531,20 @@ private String writeProjectNameIfNeeded() throws IOException {
 
         return newProjectName;
     }
+
+    // for now we simply assume that the script is called gme.groovy and already resides in the project's root directory
+    // TODO allow downloading scripts from a configured URL
```

Comment:
Agreed, lets get it working then reuse PME FileIO code? ( https://github.com/release-engineering/pom-manipulation-ext/blob/master/core/src/main/java/org/commonjava/maven/ext/core/impl/BaseGroovyManipulator.java#L94 )

Reply:
Makes perfect sense!